### PR TITLE
Add cookie info banner

### PR DESCRIPTION
**Pull request by 4D44H**
image

The EU Cookie law: http://ec.europa.eu/ipg/basics/legal/cookies/index_en.htm
-> we have first-party Cookies clearly exempt from consent according to the EU advisory body on data protection, but must still inform users that we use cookies in a popup... yeah, i know :/

Used solution: [Insights Cookie Banner](https://cookieconsent.insites.com/)

English only because: I don't care enough to translate it, and AFAIK the law leaves implementation to us.

### CODE REVIEW

**Comment by 4D44H** [comment]
[[2] Add cookie consent](https://trello.com/c/GroTl2OE/499-2-add-cookie-consent)

File: resources/views/layouts/master.blade.php [hunk 1]
```diff
@@ -1,8 +1,7 @@
 <!DOCTYPE html>
 <html>
-
 	<head>
-		<title>Lara | @yield('title', 'VedSt Default Title')</title>
+		<title>Lara | @yield('title', 'VedSt')</title>
```
Comment (stefanheyder) [inline resources/views/layouts/master.blade.php]: Why not remove the 'VedSt' entirely? None of the users know what it means

File: resources/views/layouts/master.blade.php [hunk 2]
```diff
@@ -18,6 +17,30 @@
           <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
           <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
         <![endif]-->
+        
+        <!-- EU Cookie Consent Banner -->
+        <link rel="stylesheet" 
+              type="text/css" 
+              href="//cdnjs.cloudflare.com/ajax/libs/cookieconsent2/3.0.3/cookieconsent.min.css" />
```
Comment (stefanheyder) [inline resources/views/layouts/master.blade.php]: Please don't use cdns, rather `npm install`. Here is the package https://www.npmjs.com/package/cookieconsent.

File: resources/views/layouts/master.blade.php [hunk 3]
```diff
@@ -18,6 +17,30 @@
...
+        <script src="//cdnjs.cloudflare.com/ajax/libs/cookieconsent2/3.0.3/cookieconsent.min.js"></script>
+        <script>
+            window.addEventListener("load", function(){
```
Comment (stefanheyder) [inline resources/views/layouts/master.blade.php]: Please put this in a *.ts file!

File: resources/views/layouts/master.blade.php (hunk 1 above)
Comment (P1tt187) [reply resources/views/layouts/master.blade.php]: +1

**Pull request merged**